build/build.xml

```diff
@@ -362,6 +374,12 @@
 		<delete file="${dir}/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/README.md" />
 		<delete file="${dir}/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/phpunit.xml.dist" />
 
+		<delete dir="${dir}/vendor/symfony/proxy-manager-bridge/Symfony/Bridge/ProxyManager/Tests" />
+		<delete file="${dir}/vendor/symfony/proxy-manager-bridge/Symfony/Bridge/ProxyManager/.gitignore" />
```

Comment:
Please make sure to also remove the .git folders. See related ticket for current develop: https://tracker.phpbb.com/browse/PHPBB3-13619